### DISCUSSION TITLE
[Q] any doc about how to setup serial port as console?

### DISCUSSION THREAD

**Discussion by mo-han** [zbm-dev/zfsbootmenu#528] [General]

**Comment by ahesford** [comment]
ZFSBootMenu is just a small Linux system, so you configure the console exactly as you would in your regular installation: just boot ZBM with a `console=/dev/ttyXXX` command-line argument pointing to the right serial device.

**Reply by mo-han** [reply]
thanks and good to know it's that easy.
one more thing: is USB serial converter supported at boot stage?

**Reply by ahesford** [reply]
If it works in your distribution, it will probably just work in ZFSBootMenu. However, if it requires a special kernel module, you may have to build a custom ZBM image.
